sandman/source/shell/input_window_eventful_buffer.h

```diff
@@ -4,18 +4,19 @@
 #include <cstdint>
 #include <type_traits>
 #include <string_view>
-
-#include "common/non_null.h"
+#include <cassert>
 
 namespace Shell::InputWindow { template <typename, std::size_t> class EventfulBuffer; }
```

Comment:
It may be best to remove `EventfulBuffer` and just use a wrapper around `std::string` instead, but I don't know for certain.

Reply:
Yeah, probably. And you don't really need to make the character type a template parameter, you can just hardcode it to char. Maybe NCurses supports other character types, but this is developer facing only we don't need to worry about non-English languages for this.